Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -145,6 +145,34 @@
     'suffix' : '_pixelTrackingOnly',
     'offset' : 0.5,
 }
+upgradeSteps['pixelTrackingOnlyNtupleFit'] = {
```

Comment:
I know, but I'm no longer surprised by the level of support that the Patatrack effort is receiving.

I.e., none whatesover.

Reply:
@fwyzard my proposal was intended as a way to prevent exactly this occurrence. Nobody from the patatrack side seemed to be interested. I assumed if it was important that someone would make a PR.

Reply:
OK, thank I will try to make a PR once #28281 is merged.

Reply:
@kpedro88 I see that #28281 was merged. Do you know if it entered pre11 ?

Reply:
@fwyzard no, this came after pre11 https://cmssdt.cern.ch/SDT/ReleaseNotes/CMSSW_11/CMSSW_11_0_0_pre11.html 
it will be in pre12

Reply:
Understood, thanks.

Reply:
The placeholder for the Patatrack workflows is implemented by #28522 .